lib/mapper.js

```diff
@@ -18,10 +19,25 @@ var Track = require('segmentio-facade').Track;
  */
 
 exports.identify = function(identify){
-  return {
-    token: this.settings.apiKey,
-    properties: traits(identify)
+  var opts = identify.options(this.name);
+  var msg = {
+    peopleData: {
+      token: this.settings.apiKey,
+      properties: traits(identify)
+    }
   };
+
+  // email, listId, and api_key all required
+  if (opts.listId && identify.email() && this.settings.privateKey) {
+    msg.listId = opts.listId;
+    msg.listData = {
+      email: identify.email(),
+      api_key: this.settings.privateKey,
+      confirm_optin: opts.confirmOptin != undefined ? opts.confirmOptin : this.settings.confirmOptin // default setting is true, can be overriden per event via options
```

Comment:
@sperand-io I'm interested in your thoughts on this.

I was thinking we can default this and make it easy by letting this be a setting in our UI -- BUT allow for it to be overwritten in the event that you might want to use some conditional logic and send the option via `integrations.Klaviyo.confirm_optin`. By default Klaviyo sets this to `true` so that people need to confirm via email. you can set `false` to add them right away to the list (which we need to document saying that it can be illegal/against policy to do so without explicit permission from users)